src/App/Fossa/Config/ConfigFile.hs

```diff
@@ -207,7 +207,8 @@ data ConfigFile = ConfigFile
   deriving (Eq, Ord, Show)
 
 data ConfigProject = ConfigProject
-  { configProjID :: Maybe Text
+  { configProjLocator :: Maybe Text
+  , configProjID :: Maybe Text
```

Comment:
[nit] I'd prefer not to have `Proj` abbreviated. I don't think we do that in many places IIRC.

Reply:
I don't like `Proj` either but that was the notation in data object so I just followed the convention.